### ISSUE TITLE
Allow other DAW to co-exist with pysha and pyramid

### ISSUE THREAD

**Issue by zurie** [ffont/pysha#1]
Hey Thanks for your hard work!

I am interested in extending this to work on Logic but in its current form we are at a split between pyramid and your desires for that DAW and other functionality which shares the same route buttons.  For example.   The play and Stop buttons use shepard OSC? i believe to sync / link into your DAW  but I would like to be able to . for example:  take your buttons and just send standard midi CC values which can then be basic mapped into something like Logic PRO X to control the DAW (even if basic implementations).

What I suggest is that you extend your settings.json to have a DAW in there as a global feature constant.   From there any type of functionality that could be coded for Pyramid can first check the settings for DAW of choice and then from there functionality can be built out custom for each DAW.   

This way you could maintain features and updates for pyramid but allow pysha to truly be a "External Midi Controller for Push2"    

I would love to help but my strengths are more in the interface / layout / design portion.   I am planning on forking your app and starting to work on stuff but I don't want to miss out on the features you are implementing so having some way to determine the buttons decision based on DAW would be super helpful.  This way at least I can extend and update things like your main_controls_mode.py without fear that merging it into a future branch would become impossible.

Also this could avoid the limitation of at some point pysha being so customized for pyramid that it ceases to function for other daws in its current state.

**Comment by ffont** [comment]
Hi,

Thanks for your message (and email). As I explained in the email, development of Pysha  is now abandoned because I started working on a full sequencer I call Shepherd (also open source, http://github.com/ffont/shepherd). That sequencer reuses a lot of code from Pysha and I have been refactoring it and evolving it only in Shepherd repo. When I started integrating Shepherd into Pysha I did it in this repository, and this is why you see some references to Shepherd in the Pysha repo. Earlier today I rolled back Pysha's master to the last commit before I started integrating Shepherd so it is kept as a Pyramid/MIDI controller as it was before shepherd.

I think it would be great to abstract some Pysha functionality so that it is not tied to Pyramid and it could be easily adapted for other DAWs. Unfortunately I don't have time for that. If anyone takes the challenge I'll be happy to help nevertheless.
